frontend/src/loadPostHogJS.tsx

```diff
@@ -43,6 +43,9 @@ export function loadPostHogJS(): void {
                     capture_copied_text: true,
                 },
                 process_person: 'identified_only',
+
+                // __preview_heatmaps: true,
```

Comment:
very nit: you and I know why this is commented but there might be an overlap where this is merged and someone else thinks it can be removed
```suggestion
                // commented until https://github.com/PostHog/posthog-js/pull/1131
                // __preview_heatmaps: true,
```